ckanext/dcat/harvesters/base.py

```diff
@@ -422,6 +484,20 @@ def import_stage(self,harvest_object):
         if not package_dict.get('name'):
             package_dict['name'] = self._get_package_name(harvest_object, package_dict['title'])
 
+        # copy across resource ids from the existing dataset, otherwise they'll
+        # be recreated with new ids
```

Comment:
Looked over this block of code, but I'm having trouble understanding how this prevents the resources from being recreated with new ids? Could you elaborate further for my understanding

Reply:
On line 499 the id of a resource is being set if the resource already exists.